### ISSUE TITLE
Part 2

### ISSUE THREAD

**Issue by dhammarstrom** [JohannesHaugen/Report4_Repeated_measures_studies#2]
- Are you sure about the number of participants completing the study? 17 + 17?
- Again, I think you report a mix of methods, background and results. Be more strict!
- Figure 1 contains alot of data, more than 46 data points? Don't you have issues with pseudoreplication then?
- What do you mean with: "Homogeniteten i mellom de to regresjonene ble testet ved hjelp av en enkel anova-test. Denne ga en p-verdi på 0.379. Denne var ikke statistisk signifikant og vil si at det er homogenitet i mellom de to regresjonene."?
- p-verdi på 0.1721418  Too many digits!
- What is "ges" in table 2?
- Figure 2 is nice, fix the labels!
- The same here as with report 1, p-values are hard to intepret without power analyses.

In summary: Be more stringent when writing the different parts of the report!
